srcpkgs/rssguard/template

```diff
@@ -1,18 +1,20 @@
 # Template file for 'rssguard'
 pkgname=rssguard
-version=4.1.2
+version=4.2.1
 revision=1
-build_style=qmake
-configure_args="CONFIG+=release LRELEASE_EXECUTABLE=lrelease-qt5 USE_WEBENGINE=false"
-hostmakedepends="qt5-qmake qt5-host-tools"
-makedepends="qt5-tools-devel kdeclarative-devel qt5-multimedia-devel"
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE="Release" -DUSE_WEBENGINE="false""
```

Comment:
No CMAKE_BUILD_TYPE is preferred over `Release` if possible, as it sets -O3. Was there any issues?